tests/test_capability_diagrams.py

```diff
@@ -5,23 +5,19 @@
 import pytest
 from capellambse.model.layers import ctx, oa
 
+from capellambse_context_diagrams import styling
+
+# pylint: disable-next=relative-beyond-top-level, useless-suppression
+from .conftest import SYSTEM_ANALYSIS_PARAMS  # type: ignore[import]
+
 TEST_TYPES = (oa.OperationalCapability, ctx.Capability, ctx.Mission)
 
 
-@pytest.mark.parametrize(
-    "uuid",
-    [
-        pytest.param(
-            "da08ddb6-92ba-4c3b-956a-017424dbfe85", id="OperationalCapability"
-        ),
-        pytest.param("9390b7d5-598a-42db-bef8-23677e45ba06", id="Capability"),
-        pytest.param("5bf3f1e3-0f5e-4fec-81d5-c113d3a1b3a6", id="Mission"),
-    ],
-)
+@pytest.mark.parametrize("uuid", SYSTEM_ANALYSIS_PARAMS)
 def test_context_diagrams(model: capellambse.MelodyModel, uuid: str) -> None:
     obj = model.by_uuid(uuid)
+    assert isinstance(obj, TEST_TYPES), "Precondition met"
```

Comment:
I believe this should say "Precondition failed", should it not? "Error: Precondition met" doesn't really make sense.